gpii/node_modules/lifecycleManager/src/LifecycleManager.js

```diff
@@ -252,6 +252,13 @@ var gpii = fluid.registerNamespace("gpii");
         }
     };
 
+    /** In the case we are servicing a "restore" rootAction, wrap "dangerous" promises which perform system actions so that
```

Comment:
On my system I found that the view showed 131 characters (plus one for the + in the first column) so I've tried wrapping to that, let me know if this seems readable